packages/daemon/bootstrap.js

```diff
@@ -5,6 +5,7 @@ const os = require('os');
 const { pathExists } = require('fs-extra');
 const fs = require('fs');
 const { start } = require('egg');
+const { exec } = require('child_process');
```

Comment:
Should we use execSync?

Reply:
IMO, `sync` with `await` is better.

Reply:
What you meant is `async+await`? Actually we can't use this way in this use case because the top-level await has not supported.

Reply:
The `exec` is called in an async function now.

Reply:
This is why I added this comment, it might run the migration when a user is requesting something.